docs/fenix.md

```diff
@@ -0,0 +1,58 @@
+# Fenix
+
+It is sometimes useful to test AMO with a Fenix build pointing to -dev or -stage given the tight integration between these two components. In the sequel, we describe how to construct a custom Fenix build.
```

Comment:
Is there an exact use case for a full custom build we want to call out in the docs?  (If it was just changing the collection url, we'd just change the `extensions.getAddons.browseAddons` pref via `about:addons`?)

Reply:
The install flow would only work with a custom build.

I am not sure what the pref you mentioned above does but I don't see any reference of it in AC or Fenix :/

Reply:
I filtered for `addons` in about:config and it was top of the list. Being able to define your own collection url is a thing in Fenix (and was the cause of some complication around caching, iirc).
I'm using a nightly build fwiw.

Re: the install flow, I've not tried so I'll take your word for it - I assumed it would be just changing the prefs like on desktop.

Reply:
> Being able to define your own collection url is a thing in Fenix (and was the cause of some complication around caching, iirc).

Yes but that uses a different config and not this pref, which might be coming from Fennec? On Fenix, a custom collection can be configured in the UI but only user id/collection name can be updated. The base URL isn't accessible and AC declares a default URL that cannot be modified and that isn't pref'ed. I landed a patch in Fenix to be able to override this default URL in Fenix but it is possible to do so using a build config var, not a pref.